test/e2e/specs/specs-calypso/wp-calypso-seo-preview.js

```diff
@@ -20,18 +20,15 @@ import NavBarComponent from '../../lib/components/nav-bar-component.js';
 import SidebarComponent from '../../lib/components/sidebar-component.js';
 
 const mochaTimeOut = config.get( 'mochaTimeoutMS' );
-const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
 const screenSize = driverManager.currentScreenSize();
 const host = dataHelper.getJetpackHost();
 
 describe( `[${ host }] SEO Preview page: (${ screenSize }) @parallel`, function () {
 	this.timeout( mochaTimeOut );
 	let driver;
 
-	before( 'Start browser', async function () {
-		this.timeout( startBrowserTimeoutMS );
-		driver = await driverManager.startBrowser();
-		await driverManager.clearCookiesAndDeleteLocalStorage( driver );
```

Comment:
`ensureNotLoggedIn` (now called in the shared hook) calls `clearCookiesAndDeleteLocalStorage` internally.